plugins/inputs/aliyuncms/aliyuncms.go

```diff
@@ -131,7 +139,8 @@ type (
 		PublicKeyID       string `toml:"public_key_id"`
 		RoleName          string `toml:"role_name"`
 
-		DiscoveryRegions  []string        `toml:"discovery_regions"`
+		Regions           []string        `toml:"regions"`
+		DiscoveryRegions  []string        `toml:"discovery_regions"` //this is obsolete
```

Comment:
It would be helpful to state the version making this obsolete. I'd say something like
```suggestion
		DiscoveryRegions  []string        `toml:"discovery_regions"` // obsolete since 1.19, use "regions" instead
```

Reply:
agree

Reply:
Agreed but not changed. :-)

Reply:
Forgot to push changes, now should be fixed